### PR TITLE
docs(roadmap.md): add roadmap with epics for next release #550

### LINKED ISSUE #550: docs(roadmap.md): Define a roadmap with major epics for next release
**Is your feature request related to a problem? Please describe.**
As a developer joining the project there is not a clear defined path of the epic being worked on and the target date for them

**Describe the solution you'd like**
Create a Roadmap.md to start reflecting this information

**Describe alternatives you've considered**
Work on a ReadTheDocs but there are other tasks and migrating to it should be straight-forward

**Acceptance criteria**
- [ ] Draft required epics
- [ ] Review with development team

### PR DESCRIPTION
**Pull request by petermetz**
## Dependencies

Depends on #656 (only so that the CI actually passes, not a code dependency since this is only a documentation change)

## Commit to review

Author: Peter Somogyvari <peter.somogyvari@accenture.com>
Committer: Peter Somogyvari <peter.somogyvari@accenture.com>
Date: Tue Mar 23 2021 20:41:04 GMT-0700 (Pacific Daylight Time) 

docs(roadmap.md): add roadmap with epics for next release #550

WORK IN PROGRESS
- Epics need to be created in ZenHub still

Fixes #550

Signed-off-by: Peter Somogyvari <peter.somogyvari@accenture.com>

### CODE REVIEW

**Comment by petermetz** [comment]
@sfuji822 @takeutak Kindly requesting that you review or follow-up on the review discussion within this pull request (whichever is applicable). Thank you very much in advance.

**Comment by takeutak** [comment]
@petermetz Thank you for posting. Sorry for my late reply.
As I said at Contributor's meeting, I think we need to consider the item list for V1.
I am very sorry, but I will have a holiday until next week and it is difficult for me to respond, but I plan to prepare a countermeasure plan as soon as possible after that. Could you wait for a moment, please?

**Comment by petermetz** [comment]
@takeutak 

> @petermetz Thank you for posting. Sorry for my late reply.

It's okay.

> As I said at Contributor's meeting, I think we need to consider the item list for V1.

Great. I said the same thing months ago and opened pull requests to try and move the discussion along. 
Could you please include your proposed items here in the road map and then list the ones you deem necessary for V1?

> I am very sorry, but I will have a holiday until next week and it is difficult for me to respond, but I plan to prepare a countermeasure plan as soon as possible after that. Could you wait for a moment, please?

No worries, enjoy your holiday, we can resume the conversation when you are back.

**Review by kikoncuo — APPROVED** [review]

**Comment by petermetz** [comment]
@takeutak +1

**Comment by takeutak** [comment]
@petermetz Sorry for late reviewing this PR. I added checkpoints for V1-RC on Q3 and some task items as the commit https://github.com/hyperledger/cactus/pull/716/commits/aa61ee9e824c1260cf7c38334b128e5ae4ec21d8 . Would you review it again?

File: ROADMAP.md [hunk 1]
```diff
@@ -34,17 +34,27 @@ Halves are defined as:
...
-2. HTLC plugin for Besu
+1. HTLC plugin for Besu
     * ETH
     * ERC-20
-3. Corda Open Source Ledger Connector Plugin 
+1. Corda Open Source Ledger Connector Plugin 
     * Flow Invocation
     * CordApp Deployment 
-4. Prometheus Monitoring
+1. Prometheus Monitoring
```
Comment (petermetz) [inline ROADMAP.md:43]: @takeutak Could you please fix the numbered list so that it's incrementing instead of every item having 1 as it's index?

**Comment by takeutak** [comment]
@petermetz I added a plan of checkpoints for releasing V1-RC on the roadmap. But in order to ensure the quality of the code V1-RC, I'm sure that we need to arrange the task list for error handling and test cases before fixing the roadmap.
Could I discuss with you about this topic in the next week on some meeting?

File: ROADMAP.md [hunk 2]
```diff
@@ -109,31 +109,37 @@ Halves are defined as:
...
-    - Ledger tools for Hyperledger ledgers (Fabric, Sawtooth, Besu, Indy, Iroha), Quorum, and Corda
+    - Ledger tools for Hyperledger ledgers (Fabric, Sawtooth, Besu, Indy, Iroha), Quorum, Go-Ethereum, and Corda
 
 * **Service API and Admin API**
     - SDK for BLP
     - SDK for LedgerPlugin
 
 * **Support tools**
 
+* **Dockernize**
```
Comment (petermetz) [inline ROADMAP.md]: @takeutak Typo?

**Comment by petermetz** [comment]
> @petermetz I added a plan of checkpoints for releasing V1-RC on the roadmap. But in order to ensure the quality of the code V1-RC, I'm sure that we need to arrange the task list for error handling and test cases before fixing the roadmap.
> Could I discuss with you about this topic in the next week on some meeting?

@takeutak Agreed, automated tests with sufficient coverage are very important to me as well, happy to talk about it.

**Review by takeutak — APPROVED** [review]

**Comment by petermetz** [comment]
@takeutak Could you please squash your changes into a single commit before I merge?

File: ROADMAP.md (hunk 2 above)
Comment (petermetz) [reply ROADMAP.md]: @takeutak +1

Comment (takeutak) [reply ROADMAP.md]: Sorry, I overlooked it. I will change it soon.

**Comment by takeutak** [comment]
@petermetz Thanks for comment. I will try to fix the typo and squash the commits soon. After doing it, I will contact you again.

**Comment by petermetz** [comment]
> @petermetz Thanks for comment. I will try to fix the typo and squash the commits soon. After doing it, I will contact you again.

@takeutak Thank you very much! If you need any help with the git squashing just let me know, happy to assist!

**Comment by takeutak** [comment]
> Could you please squash your changes into a single commit before I merge?
@petermetz Sorry for my late replying. I tried git squashing on this PR but failed due to the access right to your folk. Would you mind if you try to git squash?

**Comment by petermetz** [comment]
> > Could you please squash your changes into a single commit before I merge?
> 
> @petermetz Sorry for my late replying. I tried git squashing on this PR but failed due to the access right to your folk. Would you mind if you try to git squash?

@takeutak No worries, I did the squash just now. Thank you for the contributions to the roadmap!

**Pull request merged**